### DISCUSSION TITLE
GFM noteblocks proposal に対する日本語訳版の書き換え

### DISCUSSION THREAD

**Discussion by debiru** [mozilla-japan/translation#788] [General]
## 概要

2024年7月23日頃、英語版では https://github.com/mdn/content/pull/33384 によって [Notes, warnings, and callouts](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Howto/Markdown_in_MDN#notes_warnings_and_callouts) / [メモ、警告、コールアウト](https://developer.mozilla.org/ja/docs/MDN/Writing_guidelines/Howto/Markdown_in_MDN#%E3%83%A1%E3%83%A2%E3%80%81%E8%AD%A6%E5%91%8A%E3%80%81%E3%82%B3%E3%83%BC%E3%83%AB%E3%82%A2%E3%82%A6%E3%83%88) の書式がアップデートされました。

以下の MDN Style から GFM Proposal Style へ書き換えられたということです。

```
## MDN Style

> **Note:** Note!

> **Warning:** Be weary of this!

> **Callout:** Some call to action!

> **メモ:** Note!

> **警告:** Be weary of this!

> **注目:** Some call to action!

---

## GFM Proposal Style

> [!NOTE]
> Note!

> [!WARNING]
> Be weary of this!

> [!CALLOUT]
> Some call to action!
```

これに伴い、日本語版でも「メモ、警告、コールアウト」の書式書き換えが必要なのではないかと思います。以前は https://github.com/mdn/translated-content/issues/7848 で MDN Style のキーワード翻訳対応が行われました。GFM Proposal Style ではキーワードの翻訳は行われません。英語版と同じキーワードのまま動きます。

同時に、[Markdown の書き方](https://developer.mozilla.org/ja/docs/MDN/Writing_guidelines/Howto/Markdown_in_MDN#%E3%83%A1%E3%83%A2%E3%80%81%E8%AD%A6%E5%91%8A%E3%80%81%E3%82%B3%E3%83%BC%E3%83%AB%E3%82%A2%E3%82%A6%E3%83%88) ページの当該セクションの書き換えも必要かと思います。

更に、必要そうであれば https://mozilla-japan.github.io/mdn-translation-guide/ へ新規ページなりを追加して、noteblocks（メモ、警告、コールアウト）の書き方を説明することも検討します。

## 質問

- (q1) noteblocks に関して日本語版ドキュメントの全面的な書き換えを行う必要がありそうでしょうか？
- (q2) [Markdown の書き方](https://developer.mozilla.org/ja/docs/MDN/Writing_guidelines/Howto/Markdown_in_MDN#%E3%83%A1%E3%83%A2%E3%80%81%E8%AD%A6%E5%91%8A%E3%80%81%E3%82%B3%E3%83%BC%E3%83%AB%E3%82%A2%E3%82%A6%E3%83%88) ページの当該セクションを書き換える必要がありそうでしょうか？

あるいは、本件について自由にコメントをください。よろしくお願いします。

**Comment by mfuji09** [comment]
現在、GFM の noteblock スタイルにはまだ問題があるので、従来の方法を併用しています。

具体的には、 `[!NOTE]` や `![WARNING]` の次の行の先頭にリンクや引用符が来た場合、正しい書式で書いてもリンターが書式を崩してしまうので、そのような場面では従来の方法（MDN Style）を使用する必要があります。
ドキュメントの変更は行ってもよいと思いますが、現時点では従来の MDN Style も残しておく必要があると思います。

なお、上記の書き換えを行っても問題ない noteblock については、すでに本家のスタッフによって一括置換が行われています。
